release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,8 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
```

Comment:
Should we have empty release notes in here? Kind of confusing IMO

Reply:
Just thinking of the angle of a customer looking at our repo.

Reply:
I thought it would be nice for us to have a running record of what has changed to avoid the work at the end. Maybe there's a better place for it though--I can take it out.

Reply:
Maybe adding a PR template could help?

Reply:
This template would just say to mention the change in the release notes? The question is whether in the future we expect to build up releases with multiple PRs or a single PR that is tested all together and then released. In the first case, a PR template would be good.

Reply:
Yes just having a checklist would be good IMO. I mean we can remove the boxes when they don't apply but it's just a good reminder I believe.